Gopal-Kothari/Javascript/Assignment1/Ques1.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Ques1</title>
+    </head>
+    <body>
+        <center>
+            <h1>Swap Two Numbers</h1><br>
+            <hr><br>
+            <label>Enter x value :</label>
+            <input type="number" id="x" /><br><br>
+            <label>Enter y value :</label>
+            <input type="number" id="y" /><br><br>
+            <button onclick="method1()">Method 1</button>
+            <button onclick="method2()">Method 2</button>
+            <h2>Method 1 : With Temporary variable</h2>
+            <label>After swaping method2 : </label>
+            <label id="demo1"></label>
+            <h2>Method 2 : Without Temporary variable</h2>
+            <label>After swaping method2 : </label>
+            <label id="demo2"></label>
+        </center>
+        <script>
+            //with Temporary variable
+            function method1(){
+                var x = document.getElementById('x').value;
+                var y = document.getElementById('y').value;
+                var tem = x;
+                x = y;
+                y = tem;
+                document.getElementById("demo1").innerHTML = "<b> x = " + x +"<b>, y = " +y;
+            }
+            //without Temporary variable
+            function method2(){
+                var x = document.getElementById('x').value;
+                var y = document.getElementById('y').value;
+                [x,y] = [y,x];
```

Comment:
do it without arrays
x=x+y
y=x-y
x=x-y

Reply:
done